StreamChatUITestsAppUITests/Tests/Message Delivery Status/MessageDeliveryStatus+ChannelList_Tests.swift

```diff
@@ -102,7 +102,7 @@ final class MessageDeliveryStatus_ChannelList_Tests: StreamTestCase {
             userRobot.tapOnBackButton()
         }
         WHEN("participant reads the user's message") {
-            participantRobot.readMessage()
+            participantRobot.wait(0.3).readMessage()
```

Comment:
Wouldn't it be better if we make a new `readMessage(after seconds: Double = 0.3)` to avoid duplicating the same logic?

Reply:
+1

Reply:
Sure thing, done 👍

Reply:
since it already has a default value, might be good to skip passing the 0.3 everywhere. If it collides with the other `readMessage` method, might be good to rename this one.

Reply:
If we skip passing the 0.3 everywhere then it will use the function without `wait`. I removed the default value, does it look better?